add_up.rb

```diff
@@ -0,0 +1,14 @@
+3.times do 
+    print "Please enter a positive, whole number. "
+  answer = gets.chomp.to_i
+    def add_up(i)
+    j = 0
+    sum = 0
+    while j <= i do
```

Comment:
```suggestion
    while j <= i
```

Nit: You don't need a `do` after a `while` in Ruby. I suspect it's allowed because of historical reasons, but we generally don't write it this way.